core/src/test/scala/org/http4s/rho/bits/ResponseGeneratorSpec.scala

```diff
@@ -1,53 +1,52 @@
 package org.http4s.rho.bits
 
-import org.http4s.headers.{`Content-Type`, Location, `Transfer-Encoding`, `Content-Length`}
+import org.http4s.headers.{Location, `Content-Length`, `Content-Type`, `Transfer-Encoding`}
 import org.http4s._
 import org.specs2.mutable.Specification
-
-
+import scodec.bits.ByteVector
 
 class ResponseGeneratorSpec extends Specification {
   import ResponseGeneratorInstances._
 
   "ResponseGenerator" should {
     "Build a response with a body" in {
-      val result = Ok("Foo").run
+      val result = Ok("Foo").unsafeRun
```

Comment:
Could this be a property test with an Arbitrary String? If not, some code smells can get fixed up after this is merged.